src/stream/dyad_stream_api.hpp

```diff
@@ -254,7 +254,7 @@ basic_ifstream_dyad<_CharT, _Traits>& basic_ifstream_dyad<_CharT, _Traits>::
 operator= (basic_ifstream_dyad&& rhs)
 {
     m_stream = std::move (rhs.m_stream);
-    m_ctx = std::move (rhs.ctx);
+    m_core = std::move (rhs.core);
```

Comment:
`rhs.core` might be a bug. Shouldn't it be `rhs.m_core`?

Reply:
Correct. We need a test for compilation since this is a header.